data.md

```diff
@@ -34,7 +34,9 @@ Also we use `#freshId ( Int )` to generate a fresh identifier based on the eleme
 ```k
     syntax Identifier ::= #freshId ( Int )
                         | r"\\$[0-9a-zA-Z!$%&'*+/<>?_`|~=-]*" [avoid, token]
- // ------------------------------------------------------------------------
+    syntax OptionalId ::= "" [klabel(.Identifier)]
+                        | Identifier
+ // --------------------------------
```

Comment:
Leave the original horizontal rule in and put an extra blank line between them.

Reply:
Done